modules/apps/object/object-service/src/main/java/com/liferay/object/internal/system/model/listener/SystemObjectDefinitionMetadataModelListener.java

```diff
@@ -300,7 +300,16 @@ private Map<String, Object> _toDTO(T baseModel, long userId)
 			JSONObject jsonObject = _jsonFactory.createJSONObject(
 				_jsonFactory.looseSerializeDeep(object));
 
-			return jsonObject.toMap();
+			Map<String, Object> baseModelVariables =
+				baseModel.getModelAttributes();
+
+			user.getFullName();
```

Comment:
What's the purpose of that?

Reply:
Hi @rodrigopaulino because without this parameter we need chaining of method `baseModel.getModelAttributes().get("createDate")`

Reply:
Sorry @rodrigopaulino, it's wrong!